bitbots_vision/cfg/Vision.cfg

```diff
@@ -174,5 +174,6 @@ group_debug.add("vision_publish_debug_image", bool_t, 0, "Publish debug image me
 group_debug.add("ball_fcnn_publish_debug_img", bool_t, 0, "Publish the fcnn heatmap image for debug purposes", None)
 group_debug.add("vision_publish_HSV_mask_image", bool_t, 0, "Publish all three HSV color detector mask image mesages for debug purposes ", None)
 group_debug.add("vision_publish_field_mask_image", bool_t, 0, "Publish field mask image message for debug purposes", None)
+group_debug.add("caching", bool_t, 0, "DO NOT TOUCH! Used to deactivate caching for profiling reasons", None)
```

Comment:
`Do not touch` is irritating. `Used to deactivate caching for profiling reasons` is enough. Why not allow deactivation of caching in the running system (for some reasons)?

Reply:
Why even add this setting to dynamic reconfigure?

Reply:
Because it fits in the vision configuration pattern.